core/services/vrf/v1/listener_v1.go

```diff
@@ -147,6 +155,41 @@ func (lsn *Listener) Start(context.Context) error {
 	})
 }
 
+func (lsn *Listener) GetStartingResponseCountsV1(ctx context.Context) (respCount map[[32]byte]uint64, err error) {
+	respCounts := make(map[[32]byte]uint64)
+	latestBlockNum, err := lsn.Chain.Client().LatestBlockHeight(ctx)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
I think due to the way the job start/shutdown loop works, if this returns an error (which is possible if the EVM client send a request to a faulty RPC) start will return an error and the job will just be in a limbo right?

If that's the case then I think we need to wrap this with a `retry.Do` similar to what is done here: https://github.com/smartcontractkit/chainlink/blob/621576cc9d2d739e64ffbced52a2a67147af294d/core/services/vrf/delegate.go#L154-L157. If this is not the case and `Start` will get called again until it succeeds then feel free to ignore.

Reply:
I think returning an error would bubble up to the application start causing startup to fail so don't think it'd be in limbo but I agree a retry would make sense here. I've added one just like the example